packages/botonic-plugin-google-analytics-v4/README.md

```diff
@@ -0,0 +1,61 @@
+# Botonic Plugin Google Analytics
+
+## What Does This Plugin Do?
+
+With this plugin you can track the user interaction with the bot or the bot's behaviour sending the information to [Google Analytics](https://analytics.google.com/).
+
+## Setup
+
+1. Run `npm install --save @botonic/plugin-google-analytics` to install the plugin.
+2. Add it to the `src/plugins.js` file:
+
+**src/plugins.js**
+
+```javascript
+export const plugins = [
+  {
+    id: 'google_analytics',
+    resolve: require('@botonic/plugin-google-analytics'),
+    options: {
+      trackingId: 'UA-XXXXXXXX-Y', // Your Google Analytics tracking ID
+      getUserId: ({session}) => session.user.extra_data.analyticsUserId, //Optional. Method that returns a unique user ID as string
+      getUserTraits: ({session}) => { userName: session.user.extra_data.analyticsUserName, userEmail: session.user.extra_data.analyticsUserEmail }, //Optional. Method that returns an object with the user Traits
+      automaticTracking: true, //Optional. Indicates if an automatic tracking will be executed on every user interaction (true by default)
+      getEventFields: () => ({category: 'bot', action: 'user_interaction'}) //Optional. Set custom event fields to track if automatic tracking is enabled
```

Comment:
This README needs to be updated with the new GA4 parameters, right?

Reply:
yes! maybe I'll need a helping hand here ✋

Reply:
@vanbasten17 I've updated the `README` with all the changes regarding the new GA4 tracking, can you take a look to see if I'm missing something and if it looks good to you?

Reply:
LGTM! 👍  let's publish it once it's approved!